docs/metrics.md

```diff
@@ -77,4 +65,35 @@ Then navigate to the appropriate port on localhost, e.g.
 [http://localhost:9090](http://localhost:9090) for Prometheus.
 
 Ramen metrics can be found in namespace "ramen-system"; e.g. PromQL query
-`{namespace="ramen-system"}`
+`{namespace="ramen-system"}`.
+
+#### Delete stack
```

Comment:
I guess this also can be deleted, as a result of the other 2 comments.

Reply:
yes, its deleted now